airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -346,6 +347,24 @@ def transform_tidb(config: Dict[str, Any]):
         }
         return dbt_config
 
+    @staticmethod
+    def transform_firebolt(config: Dict[str, Any]):
+        print("transform_firebolt")
+        # https://docs.getdbt.com/reference/warehouse-profiles/firebolt-profile
+        dbt_config = {
+            "type": "firebolt",
+            "database": config["database"],
+            "user": config["username"],
+            "password": config["password"],
+            "schema": config["database"],
+        }
+        if "host" in config:
+            dbt_config["api_endpoint"] = config["host"]
```

Comment:
I maybe missed it somewhere, this option doesn't seem to exist on https://docs.getdbt.com/reference/warehouse-profiles/firebolt-profile ?

Reply:
You're right, looks like it was removed while I was developing this change. `api_endpoint` is redundant and I've confirmed locally supplying `host` only is sufficient. I've removed the extra config from `transform.py`